build.py

```diff
@@ -1,53 +1,53 @@
-from pathlib import Path
-
 import argparse
-import copy
+import multiprocessing
+import multiprocessing.pool
 import os
+import subprocess
+from pathlib import Path
+from typing import cast
 
+import cffsubr.__main__
 import fontmake.instantiator
 import fontTools.designspaceLib
+import fontTools.ttLib
+import fontTools.ttLib.tables._g_l_y_f as _g_l_y_f
+import psautohint.__main__
```

Comment:
it's not concerning to call `__main__` from another _command_ module, right? What if it has global state? I've never seen this pattern ("invoke a command's implementation directly when it wasn't designed as a library") before...

Reply:
I hope that it does not have relevant global state but haven't checked. I like to avoid launching subprocesses because it's so slow on Windows.